package.json

```diff
@@ -2,10 +2,26 @@
   "name": "cross-domain-utils",
   "version": "1.0.9",
   "description": "Javascript module template.",
-  "main": "index.js",
+  "main": "dist/bundle/cross-domain-utils.js",
+  "module": "dist/module/index.js",
   "scripts": {
-    "test": "gulp test",
-    "postinstall": "node postinstall.js"
+    "postinstall": "flow-typed install",
+    "build": "npm run babel & npm run webpack && npm run flow:build:fornow",
+    "webpack": "babel-node --presets es2015 ./node_modules/.bin/webpack --progress --output-path dist/bundle",
+    "babel": "babel src/ --out-dir dist/module",
+    "karma": "babel-node --presets es2015 ./node_modules/.bin/karma start",
+    "pretest": "npm run lint & npm run flow",
+    "test": "npm run karma",
+    "lint": "eslint src/ test/",
+    "flow": "flow",
+    "flow:build": "flow gen-flow-files ./src/index.js --out-dir ./dist/module",
+    "flow:build:fornow": "./node_modules/.bin/flow-copy-source ./src ./dist/module",
+    "clean": "rimraf dist coverage",
+    "reinstall": "rimraf node_modules && npm install",
+    "release:patch": "./publish.sh",
+    "release:minor": "npm run build && npm version minor",
+    "release:major": "npm run build && npm version major",
```

Comment:
Would still like to see these call `publish.sh`. You can pass in `./publish.sh major` etc.